hbase-client/src/main/java/org/apache/hadoop/hbase/client/ConnectionRegistry.java

```diff
@@ -31,11 +30,6 @@
 @InterfaceAudience.Private
 interface ConnectionRegistry extends Closeable {
 
-  /**
-   * Get the location of meta region(s).
-   */
-  CompletableFuture<RegionLocations> getMetaRegionLocations();
```

Comment:
Just removed. We trying to add api in here https://docs.google.com/document/d/1OYrCfpmmLPkSa5-AepQw8hv09zNcNPaFzSXezhcMS7E/edit?ts=5ed4e463#heading=h.kxnc4cup53tk

Reply:
Can do this in https://issues.apache.org/jira/browse/HBASE-24459